libs/flowbite-angular/src/lib/components/modal/modal.component.ts

```diff
@@ -30,11 +37,18 @@ import {
   imports: [NgClass],
   selector: 'flowbite-modal',
   template: `
-    <div [ngClass]="contentClasses().modalContainerClass">
-      <div [ngClass]="contentClasses().modalContentClass">
-        <ng-content />
+    <ng-template #modal>
+      <div class="bg-gray-900 bg-opacity-50 dark:bg-opacity-80 fixed inset-0 z-40">
```

Comment:
Update the z-index to `z-[99]`
Navbar and sidebar are something like z-40 already

Reply:
Updated to 99, but the root had z-50, so i changed it to z-[100]. Let me know if you are okay with this, or should i use z-50 and z-[49] or something like that on the backdrop.